src/App.jsx

```diff
@@ -94,44 +101,46 @@ const App = () => {
     <Layout>
       <SelectUserContext.Provider value={selectedUserObject}>
         <UserListContext.Provider value={userListObject}>
-          <Routes>
-            <Route
-              exact
-              path="/PASS/"
-              element={
-                session.info.isLoggedIn ? (
-                  <Navigate
-                    to={
-                      !localStorage.getItem('restorePath')
-                        ? '/PASS/home'
-                        : localStorage.getItem('restorePath')
-                    }
-                  />
-                ) : (
-                  <Home />
-                )
-              }
-            />
-            <Route
-              path="/PASS/home"
-              element={
-                session.info.isLoggedIn ? (
-                  <UserSection loadingUsers={loadingUsers} loadingActive={loadingActive} />
-                ) : (
-                  <Navigate to="/PASS/" />
-                )
-              }
-            />
-            <Route
-              path="/PASS/forms"
-              element={session.info.isLoggedIn ? <Forms /> : <Navigate to="/PASS/" />}
-            />
-            <Route
-              path="/PASS/inbox"
-              element={session.info.isLoggedIn ? <Inbox /> : <Navigate to="/PASS/" />}
-            />
-            <Route path="*" element={<Navigate to="/PASS/" />} />
-          </Routes>
+          <SignedInPodContext.Provider value={signedInPodMemo}>
```

Comment:
This doesn't seem to be working correctly, but we should be able to store the pod of the signed in user in a context so other components can access it.